doc/authentication-backends/JWT-authentication-module.md

```diff
@@ -0,0 +1,36 @@
+## Overview
+
+JWT authentication backend can verify JSON Web Tokens provided by the clients.
+A wide range of signature algorithms is supported, including those using public key cryptography.
+
+The module checks the signature and also the validity of the following parameters:
+
+* `exp` - an expired token is rejected,
+* `iat` - a token must be issued in the past,
+* `nbf` - a token might not be valid *yet*.
+
+Requires the SASL PLAIN method.
+
+## Configuration options
+
+* **jwt_secret_source**
+    * **Description:** A path to a file or an environment variable, which contents will be used as a JWT secret.
+    * **Warning:** A direct path to a file is read only once during startup, a path in the environment variable is read on every auth request.
```

Comment:
Direct path to a file is read only once during startup, while path in...

Reply:
Please note that while a direct path to a file is read only once during startup, a path in the environment variable is read on every auth request.

at least keep the A;)